locales-src/en-US/strings.properties

```diff
@@ -62,19 +60,6 @@ menu_action_copy_download_link=Copy Download Link
 menu_action_go_to_download_page=Go to Download Page
 menu_action_remove_download=Remove from History
 
-# LOCALIZATION NOTE (search_button): This is screenreader only text for the
-# search button.
-search_button=Search
-
-# LOCALIZATION NOTE (search_header): Displayed at the top of the panel
-# showing search suggestions. {search_engine_name} is replaced with the name of
-# the current default search engine. e.g. 'Google Search'
-search_header={search_engine_name} Search
```

Comment:
this was unused. not sure if we should track them specially or just remove as we go ?

Reply:
I'd remove them when we remove the file.

Reply:
How does removal affect migration recipes? Seems like the recipes need to access the original property value, but I suppose given the cross-channel nature of gecko-strings, mozilla-beta still having the properties would make them available ?